packages/otel/src/aws-instrumentation-factory.ts

```diff
@@ -0,0 +1,7 @@
+import { AwsInstrumentation } from "@opentelemetry/instrumentation-aws-sdk";
+
+export const createAwsInstrumentation = () => {
+  return new AwsInstrumentation({
+    suppressInternalInstrumentation: true,
```

Comment:
what does it mean? can you add comment?

Reply:
Done in [`3691b21` (#1758)](https://github.com/saleor/apps/pull/1758/commits/3691b21c7e401adacc44f9af48ede1e77f96611a)